syntax = "proto3";

import "auth.proto";

package foobar.tokenizer;

message Token {
    string username = 1;
    string token = 2;
}

service TokenDispenser {
    rpc create_token(auth.Auth) returns (Token) {}
}